frontend/src/components/routes.tsx

```diff
@@ -277,7 +277,9 @@ export const APP_ROUTES: IRouteEntry[] = [
         routeVisibility(true, [Feature.GetQuotas], ['canListQuotas'])
     ),
 
-    MakeRoute<{}>('/connect-clusters', KafkaConnectOverview, 'Connectors', LinkIcon, true),
+    MakeRoute<{}>('/connect-clusters', KafkaConnectOverview, 'Connectors', LinkIcon, true,
+        routeVisibility(true, [Feature.PipelineService])
```

Comment:
If you do this `routeVisibility(true, [Feature.PipelineService])`, this will disable the connect overview page when the pipeline service is not available.

At first this sounds like it is intended. 
But think about the case where someone has the pipeline service disabled, but still wants to see what is going on with the old/basic kafka connect.

I think you don't want to add this routeVisibility filter.
The `KafkaConnectOverview` page is an exception in that it is aware that it needs to deal with 2 different apis (rp connect aka pipelines, and kafka connect) and has automatic handling for this.

Reply:
Your removal of that line from `routesIgnoredInServerless` should be enough for the intended effect (unless I'm missing something)

Reply:
Thanks Moritz! Fixed.